trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -67,6 +67,7 @@ const submitDealAfterUkefIds = async (dealId, dealType, checker) => {
 
   const submittedDeal = await api.submitDeal(dealType, dealId);
   const mappedDeal = mapSubmittedDeal(submittedDeal);
+  console.log(mappedDeal);
```

Comment:
leave this one for now, but create a separate PR to remove it

Reply:
added to my bug report